packages/@react-aria/checkbox/src/useCheckboxGroup.ts

```diff
@@ -11,13 +11,15 @@
  */
 
 import {AriaCheckboxGroupProps} from '@react-types/checkbox';
+import {checkboxGroupNames} from './utils';
+import {CheckboxGroupState} from '@react-stately/checkbox';
 import {filterDOMProps, mergeProps} from '@react-aria/utils';
 import {HTMLAttributes} from 'react';
 import {useLabel} from '@react-aria/label';
 
 interface CheckboxGroupAria {
   /** Props for the checkbox group wrapper element. */
-  checkboxGroupProps: HTMLAttributes<HTMLElement>,
+  groupProps: HTMLAttributes<HTMLElement>,
```

Comment:
while I understand the sentiment to match against the returned `role`, isn't this confusing (DX-wise)? given that `useRadioGroup` returns `radioGroupProps`?

Reply:
Well, in that case the role is `radiogroup`.

Reply:
Right, the `role` is mainly opaque to users though - the point of React Aria is so people don't have to worry about this. Seems like one might not really know what exact role is being prefilled by React Aria in the returned objects, but they certainly know how the component that they have just used is named.

I don't care about this as much as TS will help me with this anyway but I kinda find the choice somewhat odd as the chosen name focuses on the low-level implementation detail that is hidden from the user, rather than on a high-level feature that is actually used directly by them.